docs/OnTrack/Deployment/Google Cloud/Overview.md

```diff
@@ -0,0 +1,69 @@
+# Google Cloud - Overview
+
+[Overview](#overview)
+
+[Aims](#aims)
+
+[Initial stages](#initial-stages)
+
+[Next stages](#next-stages)
+
+[Links](#links)
+
+[Assets](#assets)
+
+## Overview
+
+![Google Cloud - Doubtfire](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/GoogleCloud_Doubtfire.jpg "Google Cloud - Doubtfire")
+
```

Comment:
I think it would be good to address why this project exists in the overview - so Thoth Tech has their own deployment of Ontrack to develop rather than the Deakin deployment

Reply:
Will help future teams understand and make decisions in regards to the project

Reply:
I've updated the Google Cloud overview to cover some background of the deployment team project objectives.